### PR TITLE
5pm-1 Added herokuqa link to readme

### LINKED ISSUE #10: Set up team deployments for qa on Heroku
# Acceptance Criteria:

- [ ] The main branch is deployed on Heroku at, for example,
      <https://s22-5pm-1-courses-qa.herokuapp.com>, and is set up for automatic
      deployment.  
- [ ] All members of the team are added to the Heroku deployment.
- [ ] The course instructor (`phtcon@ucsb.edu`) and the mentor listed
      at <https://ucsb-cs156.github.io/s22/info/teams/> is added to
      the heroku deployment.

### PR DESCRIPTION
**Pull request by alexward120**
In this pull request, I updated the README file to show the herokuqa dashboard link and herokuqa link for 5pm-1

### CODE REVIEW

**Review by chrisstasiowski01 — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses/branch/main/graph/badge.svg?token=LvEIQ6tYti)](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses)
 
+* Team 1 Heroku QA Link: <https://s22-5pm-1-courses-qa.herokuapp.com/>
+
```
Comment (andrewpengucsb) [inline README.md:7]: Missing a link to the heroku dashboard for the qa deployment. Make sure it is shared with all of the staff as well. Also please add a description to the pull request. Thanks!

**Review by andrewpengucsb — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (alexward120) [reply README.md:7]: Changes fixed

Comment (andrewpengucsb) [reply README.md:7]: Still missing a link to the dashboard and pull request description

Comment (alexward120) [reply README.md:7]: Ok i think i fixed it now

**Review by andrewpengucsb — APPROVED** [review]
LGTM

**Review by nthnga13 — APPROVED** [review]
LGTM

**Pull request merged**